includes/reader-activation/sync/class-esp-sync.php

```diff
@@ -195,6 +195,9 @@ public static function scheduled_sync( $user_id, $context ) {
 	 * @return array|\WP_Error The contact data or WP_Error.
 	 */
 	public static function get_contact_data( $user_id ) {
+		if ( ! class_exists( '\WC_Customer' ) ) {
+			return;
+		}
```

Comment:
Per the method docblock, it's best to return a WP_Error here

Reply:
Good call, updated!